openapi.yaml

```diff
@@ -143,374 +82,417 @@ paths:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person or job not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/ssn:
+            $ref: '#/definitions/JobResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/ssn:
     post:
-      description: Create person's SSN
+      description: Create a person's SSN
       tags:
         - ssn
-      operationId: postSSN
+      operationId: postSsn
       parameters:
-        - name: ssnBody
+        - name: body
           in: body
-          description: Result object for an SSN
+          description: SSN post body
           required: true
           schema:
-            $ref: '#/definitions/NewSSNResultObject'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/pretty'
-        - $ref: '#/parameters/authorization'
+            $ref: '#/definitions/SsnPostBody'
+        - $ref: '#/parameters/osuId'
       responses:
-        202:
+        '202':
           description: SSN record accepted.
           schema:
-            $ref: '#/definitions/SSNResultObject'
-        400:
-          description: Bad request
-          schema:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/image:
+            $ref: '#/definitions/SsnResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/image:
     get:
-      description: |
-        Get an ID card image for a person. If the person exists but doesn't have
-        an ID card image, a default image will be returned.
+      description: Get a person's image
       tags:
         - images
-      operationId: getImage
+      operationId: getImageById
       produces:
         - image/jpeg
         - application/json
       parameters:
+        - $ref: '#/parameters/osuId'
         - in: query
           name: width
           description: Resizes image to a pixel value for width while keeping original aspect ratio
           required: false
           type: integer
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/pretty'
-        - $ref: '#/parameters/authorization'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
             type: file
           headers:
-              Content-type:
-                type: string
-                description: image/jpeg
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/meal-plans:
+            Content-type:
+              type: string
+              description: image/jpeg
+        '404':
```

Comment:
Also need a 400 and a 500.

Reply:
What can cause a 400 here?

Reply:
If `width` is out of bound or not an integer.